velox/expression/tests/SpecialFormRewriteTestBase.cpp

```diff
@@ -33,21 +33,29 @@ void SpecialFormRewriteTestBase::TearDown() {
   expression::ExprRewriteRegistry::instance().clear();
 }
 
+void SpecialFormRewriteTestBase::testRewrite(
+    const core::TypedExprPtr& expr,
+    const core::TypedExprPtr& expected,
+    const RowTypePtr& type) {
```

Comment:
type is not used in the function. Please remove the argument.

Reply:
type is required by other tests in `ExprOptimizerTest.cpp`, the variant of `testRewrite` that takes in `TypedExpr` was added because Duckdb parser has a limitation that it cannot parse IN expressions with non constant IN-list.